mediapipe/tasks/python/benchmark/vision/image_classifier/setup.sh

```diff
@@ -0,0 +1,6 @@
+# Install Python dependencies.
```

Comment:
Note that I removed this file as part of the import. I changed your script to build via Bazel (added a BUILD file). I moved the commands to download the models to the README. Building via Bazel allows us to ensure that these script don't break as we change our CI.

Reply:
Agreed and it ensures we're pulling the updated models while benchmarking.